conf/nginx-docker/nginx.conf

```diff
@@ -29,7 +29,7 @@ server {
 	# Product Opener needs a root domain + a wildcard for all subdomains
 	server_name ${PRODUCT_OPENER_DOMAIN} *.${PRODUCT_OPENER_DOMAIN};
 
-	access_log /var/log/${productopener_access_file_prefix}access.log;
+	access_log /var/log/nginx/${productopener_access_file_prefix}access.log;
```

Comment:
Shan't we also add an explicit error_log location ?

Reply:
Well by default error_log is in /var/log/nginx/ 
I think here you wanted to add "static_" to the access log, do you want to add it to the error log too?

Reply:
No it's ok :-)